### DISCUSSION TITLE
Финальные классы компонентов

### DISCUSSION THREAD

**Discussion by finagin** [moonshine-software/moonshine#1177] [General]
Почему классы компонентов типа `Profile` финальные? Можно расширить `view` компонента через `resources/views/vendor/moonshine`, но класс расширить нельзя, потому что он финальный

**Comment by lee-to** [comment]
Очень простые классы, которые проще сделать без расширения, если есть аргументы то можно без дискуссии создавать пр с удалением модификатора final
